src/contexts/EnvironmentContext.js

```diff
@@ -28,6 +29,20 @@ export const EnvironmentProvider = ({ children }) => {
             appbarLogo: process.env.REACT_APP_APPBAR_LOGO,
             loginLogo: process.env.REACT_APP_LOGIN_LOGO,
             brandingName: process.env.REACT_APP_BRANDING_NAME,
+
+            homeTitle: "Home",
+            zonesTitle: "Zones",
+            serversTitle: "Servers",
+            resourcesTitle: "Resources",
+            usersTitle: "Users",
+            groupsTitle: "Groups",
+            specificQueriesTitle: "Specific Queries",
+
+            pageTitle: "",
```

Comment:
is this necessary just to instantiate the variable?

Reply:
Ohhh that's a really good point. Just tested it without instantiating the `pageTitle` property, and it worked fine (since we're running `environment.pageTitle = ...`). Thus, I think it's just a matter of styling. I personally think that it's easier to read and understand when we initialize `pageTitle` to an empty string (as the `titleFormat` method uses the `pageTitle` property), but I can also see how the code would be cleaner without it.

Reply:
happy to have it there - maybe leave a comment that it's a placeholder that gets filled by other methods.